pom.xml

```diff
@@ -295,6 +295,18 @@
         <enabled>false</enabled>
       </snapshots>
     </repository>
+    <!-- TODO: This can be deleted after Spark 1.5 is posted -->
```

Comment:
I'm actually not sure -- I followed Patrick's last pr of this kind.

What happened without this is that Mima would fail and complain about all classes we marked as exclude for 1.5. I actually don't know exactly how mima works. Adding this doesn't seem to help much either.